ui/src/data/containers/DetailPageContainer/DetailPageContainer.tsx

```diff
@@ -55,74 +57,102 @@ const DetailPage = ({
   return (
     <>
       <DocumentHead title={title.get('title')} description={metaDescription} />
-      <Row justify="center" data-testid="data-detail-page-container">
-        <Col className="mv3" xs={24} md={22} lg={21} xxl={18}>
-          <ContentBox
-            className="sm-pb3"
-            leftActions={
-              <>
-                {urls && (
-                  <UrlsAction
-                    urls={urls}
-                    text="links"
-                    page="Data detail"
-                    trackerEventId="Data website"
-                  />
-                )}
-                {isSuperUserLoggedIn && (
-                  <APIButton url={window.location.href} />
-                )}
-              </>
-            }
-            rightActions={
-              citationCount !== null && citationCount !== undefined ? (
-                <IncomingLiteratureReferencesLinkAction
-                  itemCount={citationCount}
-                  referenceType="citation"
-                  linkQuery={getReferencingPapersQueryString(recordId)}
-                  trackerEventId="Citations link"
-                  eventCategory="Data search"
-                />
-              ) : (
-                <></>
-              )
-            }
+      <Row
+        className="__DetailPage__"
+        justify="center"
+        data-testid="data-detail-page-container"
+      >
+        <Col xs={24} md={22} lg={21} xxl={18}>
+          <Row className="mv3" justify="center" />
+          <Row
+            className="mv2"
```

Comment:
we can put that mv3 for consistency with the literature detail https://github.com/inspirehep/inspirehep/pull/3465#issuecomment-2687200488